### PR TITLE
Aded participation zone database structure

### PR DESCRIPTION
**Pull request by alecslupu**
This PR adds the required models for Zone and Boxes.
Ref #10

### CODE REVIEW

File: app/models/decidim/participatory_documents/zone.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      include Decidim::Traceable
+      include Decidim::Loggable
+      include Decidim::TranslatableResource
+      include Decidim::TranslatableAttributes
+      include Decidim::Publicable
+      belongs_to :document, class_name: "Decidim::ParticipatoryDocuments::Document"
+      has_many :annotations, class_name: "Decidim::ParticipatoryDocuments::Annotation", dependent: :destroy
+
+      translatable_fields :title, :description
+      POSSIBLE_STATES = %w(draft published private closed).freeze
```
Comment (microstudi) [inline app/models/decidim/participatory_documents/zone.rb:15]: I think we should have three different attributes, published or not (as stated by the `Decidim::Publicable`), private/not private and closed (with `closed_at` datetime attribute) .
In any case, is this used anywhere?

**Review by microstudi — CHANGES_REQUESTED** [review]
Thank @alecslupu 
I've only found a change in the attributes for the zone model.

File: app/models/decidim/participatory_documents/zone.rb (hunk 1 above)
Comment (alecslupu) [reply app/models/decidim/participatory_documents/zone.rb]: @microstudi i have removed this so we can tackle in a separate PR.

Comment (microstudi) [reply app/models/decidim/participatory_documents/zone.rb]: > Can we have combinations like: Published & private ? Private & closed?

Yes, we can have those combinations.

**Pull request merged**